lib/strategies/android-chrome.js

```diff
@@ -9,6 +10,26 @@ function AndroidChromeStrategy(options) {
 AndroidChromeStrategy.prototype = Object.create(BaseStrategy.prototype);
 AndroidChromeStrategy.prototype.constructor = AndroidChromeStrategy;
 
+BaseStrategy.prototype._attachListeners = function () {
```

Comment:
Also, you're overwriting the BaseInput prototype, not the AndroidChrome Strategy prototype